nexus/db-queries/src/db/datastore/deployment.rs

```diff
@@ -4661,37 +4691,30 @@ mod tests {
         }
     }
 
-    // Verify that every blueprint-related table contains ≥1 row for `blueprint_id`.
+    // Verify that every blueprint-related table contains ≥1 row across test blueprints.
     // Complements `ensure_blueprint_fully_deleted`.
     async fn ensure_blueprint_fully_populated(
-        datastore: &DataStore,
-        blueprint_id: BlueprintUuid,
+        _datastore: &DataStore,
+        cumulative_counts: &BlueprintTableCounts,
```

Comment:
I feel like we may as well make this a method on `BlueprintTableCounts` at this point.